.sage/main.go

```diff
@@ -32,10 +32,9 @@ func DoubleGitCheck(ctx context.Context) error {
 }
 
 func Default(ctx context.Context) error {
-	sg.Deps(ctx, ConvcoCheck, GoLint, GoTest, FormatMarkdown, FormatYaml, BackstageValidate)
-	sg.SerialDeps(ctx, GoModTidy)
-	sg.SerialDeps(ctx, GoLicenses, GitVerifyNoDiff)
-	return nil
+	sg.Deps(ctx, DoubleGitCheck)
+	cmd := sggit.Command(ctx, "status", "--short")
```

Comment:
`DoubleGitCheck` wont return any changes, but `git status --short` verifies that files actually were changed.